security-controls/sast.py

```diff
@@ -0,0 +1,315 @@
+"""Update SAST security-controls data."""
+
+import logging
+import os
+import re
+
+import requests
+from google.cloud import firestore
+from requests.auth import HTTPBasicAuth
+
+SECURITY_CONTROLS = os.environ['SC_FIRESTORE_COLLECTION']
+SAST_DETAILS = os.environ['SAST_FIRESTORE_COLLECTION']
```

Comment:
If the information stored in that collection does not help the cronjob to run correctly, I don't think we should store more information. In case we need that information in the future for another task, then we can add it again.

Reply:
I want it for SAST management. It's well tested.

Reply:
Note I removed Codacy user info as discussed separately.

Reply:
But I think if we are going to add SAST management will add it again, now we don't have it as part of this ticket/roadmap.

Reply:
That collection contains information for repositories that are not part of Terra services/security controls.

Reply:
Moved to a separate branch.